src/status.rs

```diff
@@ -458,6 +464,52 @@ impl StatusBuilder {
         self
     }
 
+    pub fn spacefed(mut self, spacefed: Spacefed) -> Self {
+        self.spacefed = Some(spacefed);
+        self
+    }
+
+    pub fn add_event(mut self, event: Event) -> Self {
+        if let Some(ref mut events) = self.events {
+            events.push(event);
+        } else {
+            self.events = Some(vec![event]);
+        }
```

Comment:
FWIW: This would work, but it isn't any nicer...
```Rust
self.radio_show.as_mut()
    .map(|vec| vec.push(radio_show))
    .or_else(|| { self.radio_show = Some(vec![radio_show]); None });
```
Maybe we should just replace the Option<Vec> with just Vec and not serialize it when empty.